backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -53,21 +53,4 @@
     <% end %>
 
   </div>
-
-  <div data-hook="admin_user_form_password_fields" class="col-6">
-    <%= f.field_container :password do %>
-      <%= f.label :password %>
-      <%= f.password_field :password, class: 'fullwidth' %>
-      <%= f.error_message_on :password %>
-    <% end %>
-
-
-    <% if f.object.respond_to?(:password_confirmation) %>
-      <%= f.field_container :password do %>
-        <%= f.label :password_confirmation %>
-        <%= f.password_field :password_confirmation, class: 'fullwidth' %>
-        <%= f.error_message_on :password_confirmation %>
-      <% end %>
-    <% end %>
-  </div>
```

Comment:
I think we should keep these fields and only display them

```rb
if @user == try_spree_current_user
```

also, we should add a "current password" field. Then we should have addressed all security concerns.